dotnet/samples/TeamsChefBot/teamsapp.local.yml

```diff
@@ -0,0 +1,82 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.1.0/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
```

Comment:
The js sample has yml file with polished comment. Better to keep consistence with that.

Reply:
It seems that some comments are for TTK VSC only.

Reply:
Updated.